### PR TITLE
Added note box about default sccs to architecture guide

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1577830

### CODE REVIEW

**Comment by bfallonf** [comment]
cc @enj 

Will update the BZ.

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -525,6 +525,25 @@ privileged         true      [*]       RunAsAny    RunAsAny           RunAsAny
...
 
+ifdef::openshift-origin,openshift-enterprise[]
+[IMPORTANT]
+====
+Do not modify the default SCCs. Customizing the default SCCs can lead to issues
+when {product-title} is upgraded. Instead,
+xref:../../admin_guide/manage_scc.adoc#creating-new-security-context-constraints[create
+new SCCs].
+====
+endif::openshift-enterprise,openshift-origin[]
```
Comment (vikram-redhat) [inline architecture/additional_concepts/authorization.adoc]: @bfallonf - brice, wrong endif (or wrong ifdef :P). Can you please fix it today?

Comment (bfallonf) [reply architecture/additional_concepts/authorization.adoc]: @vikram-redhat fixed in #9519

**Pull request merged**